fuse/direntry.go

```diff
@@ -30,6 +30,24 @@ func (d DirEntry) String() string {
 	return fmt.Sprintf("%o: %q", d.Mode, d.Name)
 }
 
+// DirEntryIno is a DirEntry struct extended by the "Ino" field for the
+// inode number.
+type DirEntryIno struct {
```

Comment:
I mean, I'm good with that, but I would not propose it without consulting with you first!